packages/shared/src/dto/workflows/workflow-deprecated.dto.ts

```diff
@@ -0,0 +1,29 @@
+import { IWorkflowStepMetadata } from '../../entities/step';
+import { BuilderFieldType, BuilderGroupValues, FilterParts } from '../../types';
+import { MessageTemplateDto } from '../message-template';
+
+export class StepVariantDto {
+  id?: string;
+  _id?: string;
```

Comment:
I was hoping to not see this variant ever again, why are we exposing it in all it's glory? is it really nessesery, what we expose is harder to remove in the future, I would like to not have this exposed, it's repetatice and super confusing

Reply:
It was already exposed, I didn't added this I only renamed the file.

Its used in legacy (v1) workflow endpoints and usecases.

Reply:
resolving, it wasn't clear from git